pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -123,28 +124,31 @@ def map_fips(self, fips):
         all_hospitalized_today = None
         try:
             fit_results = load_inference_result(fips)
+
+
+        # Fit results not always available if the fit failed, or there are no
+        # inference results.
         except (KeyError, ValueError):
             fit_results = None
             logging.warning(f'Fit result not found for {fips}: Skipping inference elements')
 
         for i_policy, suppression_policy in enumerate(policies):
-            if suppression_policy == 'suppression_policy__full_containment':  # No longer shipping this.
-                continue
-            if suppression_policy == 'suppression_policy__inferred' \
-                    and len(fips) == 5 \
-                    and fips not in self.df_whitelist.fips.values:
+
+            # Don't ship full containment, and only ship inference for
+            # whitelisted counties.
+            if suppression_policy == 'suppression_policy__full_containment' \
+                    or (suppression_policy == 'suppression_policy__inferred'
+                        and len(fips) == 5
+                        and fips not in self.df_whitelist.fips.values):
                 continue
 
             output_for_policy = pyseir_outputs[suppression_policy]
             output_model = pd.DataFrame()
 
+            # Hospitalizations need to be rescaled by the inferred factor to
+            # match observations for display purposes.
             if suppression_policy == 'suppression_policy__inferred' and fit_results:
-                if len(fips) == 5 and fips not in self.df_whitelist.fips.values:
```

Comment:
This is moved to the above location to include counties that don't have fit results.